main.go

```diff
@@ -15,6 +16,6 @@ var (
 func main() {
 	cmd.SetVersion(fmt.Sprintf("%s, commit %s, date %s", version, commit, date))
 	if err := cmd.Execute(); err != nil {
-		log.WithError(err).Fatal("Command aborted.")
+		os.Exit(1)
 	}
```

Comment:
Is there still an error being printed in error cases?

Reply:
Yes, Cobra prints the errors already